docs/source/en/quantization/bitsandbytes.md

```diff
@@ -416,6 +416,51 @@ text_encoder_2_4bit.dequantize()
 transformer_4bit.dequantize()
 ```
 
+## torch.compile compatibility
+
+<hfoptions id="bnb">
+<hfoption id="8-bit">
+
+You can speed up inference of `bitsandbytes`-quantized models with `torch.compile()`. Make sure you have
+the latest `bitsandbytes` installed - `pip install -U bitsandbytes`. We also recommend installting PyTorch
+nightlies. Follow instructions [here](https://pytorch.org/).
```

Comment:
I would place this outside the `<hfoptions>` blocks and under `## torch.compile` since it applies to both 8 and 4-bit.

```suggestion
Speed up inference with `torch.compile`. Make sure you have
the latest `bitsandbytes` installed and we also recommend installing [PyTorch nightly](https://pytorch.org/get-started/locally/).

```py
pip install -U bitsandbytes
```
```